### ISSUE TITLE
thread(handle_client_function, clientConnection).detach();

### ISSUE THREAD

**Issue by liangjiaru** [j-r-l/coding_practice#71]
- is it concurrent enough?
  - more threads in handle_client() ;
